test/BaselineOutput/Common/EntryPoints/core_manifest.json

```diff
@@ -15780,17 +15772,10 @@
                   "Type": {
                     "Kind": "Enum",
                     "Values": [
-                      "Default",
                       "DefaultValue",
-                      "Def",
                       "Mean",
-                      "Min",
                       "Minimum",
-                      "Max",
-                      "Maximum",
-                      "SpecifiedValue",
-                      "Val",
-                      "Value"
```

Comment:
All three of them are marked as hidden. Do you know if that is intentional?

Reply:
Hi @sfilipi thanks for the question. Hiding the "abbreviated" versions `Val`, `Value`, I think is probably intentional. `SpecifiedValue` though is an interesting choice. I am not certain whether hiding *all* options related to this is intentional, though assuredly it was done.

https://github.com/dotnet/machinelearning/blob/c023727b76970ab913ec1ce38276508835c17bcf/src/Microsoft.ML.Transforms/NAReplaceTransform.cs#L56-L57

@yaeldekel wrote this enum, but that was back in 2016. She may not remember, but just tagging her on the off chance she does.

Reply:
Should probably expose the "SpecifiedValue". Checking the NA Replace transform, if the enum is not set to that, the value set for the ReplacementString is ignore. 

---
In reply to: [195274816](https://github.com/dotnet/machinelearning/pull/356#discussion_r195274816) [](ancestors = 195274816)

Reply:
Hi @sfilipi I agree with that logic. I will unhide.